tests/e2e/vsphere_volume_expansion.go

```diff
@@ -651,6 +651,8 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 		var originalSizeInMb, fsSize int64
 		var err error
 		var expectedErrMsg string
+		vcAddress := e2eVSphere.Config.Global.VCenterHostname + ":" + sshdPort
+		featureEnabled := isFssEnabled(vcAddress, "CNS_NEW_SYNC")
```

Comment:
It is better if we read CNS_NEW_SYNC as a constant. please add it in e2e_common.go